lldb/tools/driver/Driver.cpp

```diff
@@ -423,6 +426,64 @@ SBError Driver::ProcessArgs(const opt::InputArgList &args, bool &exiting) {
   return error;
 }
 
+#ifdef _WIN32
+inline std::wstring GetPathToExecutableW() {
+  // Iterate until we max out the Windows max path length (256*2^7 > 32,767).
+  static const size_t maximumIterations = 7;
+  std::wstring buffer;
+  DWORD bufferSize = MAX_PATH;
+  for (size_t i = 0; i < maximumIterations; i++) {
+    buffer.resize(bufferSize);
+    DWORD length = GetModuleFileNameW(NULL, &buffer[0], bufferSize);
+    if (length < buffer.length()) {
+      buffer.resize(length);
+      return buffer;
+    }
+    bufferSize *= 2;
+  }
+  return L"";
+}
+
+// Resolve the Embeddable Python directory bundled with the Swift toolchain.
+// If it exists, add it to the list of DLL search directories.
+// The installation of Embeddable Python is optional when installing the Swift
+// toolchain, therefore the directory might not exist.
+void AddPythonDLLToSearchPath() {
+  std::wstring modulePath = GetPathToExecutableW();
+  if (modulePath.empty()) {
+    WithColor::error() << "Failed to get module file path. Error: "
+                       << GetLastError() << "\n";
```

Comment:
```suggestion
                       << GetLastError() << '\n';
```

Note that `"\n"` would cause unnecessary buffering.